package.json

```diff
@@ -8,9 +8,9 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
+    "start": "copyfiles --flat src/config/templates/*.hbs dist/config/templates && nest start",
```

Comment:
Não acho muito legal usar isto, se houver outras necessidade parecidas isso vai ser a porta de entrada para aumentar muito mais, além de forçar alterações no CI/CD das plataformas envolvidas. Minha sugestão é sim trabalhar com .hbs mas ler o layout/template de algum repositório, criarmos um repositório pra isso e dependendo alguma outra configuração.

Reply:
Consegui tirar